spack-scripting/tests/test_nalu_wind_nightly.py

```diff
@@ -0,0 +1,5 @@
+import pytest
+import importlib
+
+nalu_wind_nightly = importlib.import_module('...repos.exawind.packages.nalu-wind-nightly.package', 'package')
+#from ...repos.exawind.packages.nalu-wind-nightly.package import *
```

Comment:
```suggestion
from spack.pkg.exawind.nalu_wind_nightly import NaluWindNighlty 
```
Have you tried this?